lib/sqlalchemy/orm/relationships.py

```diff
@@ -1923,8 +1923,8 @@ def _add_reverse_property(self, key):
         if other.viewonly:
             util.warn_limited(
```

Comment:
**mike bayer** (zzzeek) wrote:

so if we are doing deprecated, this should be warn_deprecated().  however, we then have to make a choice about having specific info in the message.   a regular warnings.warn() will be added to an unbounded growth set inside of the warnings lib, so we would have to remove specific strings.  OTOH, warn_limited does a modulus trick so that it only warns up to 10 times, but this feature is not available in warn_deprecated().   so we can either remove the "%s" tokens from the string here, or we can add the "limited" capability to warn_deprecated().  probably the latter as it will be difficult for people to otherwise know where this warning is originating from.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1891